tests/shared/src/test/scala/org/typelevel/ci/CIStringSpec.scala

```diff
@@ -79,6 +80,30 @@ class CIStringSpec extends Specification with Discipline {
     }
   }
 
+  "isEmpty" >> {
+    "is true given an empty string" >> {
+      isEmpty(CIString(""))
```

Comment:
Hmm, now I'm wondering whether it would have been a better idea to define `isEmpty` and `nonEmpty` in the class and not the companion object...

Reply:
I would, yes.